tst/testinstall/package.tst

```diff
@@ -222,6 +223,35 @@ gap> info := rec(
 >     PackageName := "pkg",
 >     Subtitle := "desc",
 >     Version := "0",
+>     Date := "01/20/2015",
```

Comment:
Shouldn't there also be a test with an ISO date?

Reply:
You're absolutely right especially since there is something wrong that I only spotted because I did the test. Sorry should have already done that!